index.php

```diff
@@ -1 +1,247 @@
+<?php
+$time_start = microtime( true );
+mb_internal_encoding( "UTF-8" );
+if ( isset( $_POST['data'] ) ) {
+	$numberOfCharacter                                    = strlen( $_POST['data'] );
```

Comment:
Please change code formatting setup a little:
https://www.php-fig.org/psr/psr-12/#24-indenting
`Code MUST use an indent of 4 spaces for each indent level, and MUST NOT use tabs for indenting.`